src/package_io/input_parser.star

```diff
@@ -786,6 +787,7 @@ def default_network_params():
         "electra_fork_epoch": 100000000,
         "eip7594_fork_epoch": 100000001,
         "eip7594_fork_version": "0x70000038",
+        "eof_activation_epoch": "",
```

Comment:
```suggestion
        "eof_activation_epoch": "100000002",
```

Reply:
No, as that can break ELs that don't have the praugeEOFTime defined yet.

Reply:
ugh